sources/osgShader/node/skeletal.js

```diff
@@ -6,12 +6,12 @@ define( [
 ], function ( MACROUTILS, Texture, ShaderUtils, Node ) {
     'use strict';
 
-    var AnimationNode = function () {
+    var SkeletalNode = function () {
```

Comment:
same here, it should be named skinning node to be coherent with the other part of the code